src/styles.css

```
@@ -0,0 +1,68 @@
+body,
+p,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  line-height: 14.6px;
+  background-color: antiquewhite;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  background-color: #fff;
+}
+
+.logo img {
```

```suggestion
.logo_img {
```
Consider using a class selector for your image